lib/booktype/apps/reader/templates/reader/book_full_view.html

```diff
@@ -3,69 +3,90 @@
 
 {% block extra_header %}
   {% compress js %}
+    <script type="text/javascript" src="{% static 'core/js/jquery.mCustomScrollbar.concat.min.js' %}"></script>
     <script type="text/javascript" src="{% static 'edit/js/booktype/utils.js' %}"></script>
     <script type="text/javascript" src="{% static 'reader/js/fullview.js' %}"></script>
   {% endcompress %}
+  {% compress css %}
+    <link href='{% static 'core/css/jquery.mCustomScrollbar.min.css' %}' rel='stylesheet' type='text/css'/>
+    <link href='{% static 'reader/css/fullview.css' %}' rel='stylesheet' type='text/css'/>
+
+
+    {% if theme.active != 'custom' %}
+      <link href='{% static 'themes/'|add:theme.active|add:'/preload.css' %}' rel='stylesheet' type='text/css'/>
+      <link href='{% static 'themes/'|add:theme.active|add:'/editor.css' %}' rel='stylesheet' type='text/css'/>
+    {% else %}
+      <script type="text/javascript">
+        var themeCustom = JSON.parse('{{ theme.custom|safe }}');
+      </script>
+      <link href='{% static 'themes/'|add:theme.active|add:'/editor.css' %}' rel='stylesheet' type='text/css'/>
+      <style id="embeduserstyle" type="text/css"></style>
+    {% endif %}
+
+  {% endcompress %}
 {% endblock %}
 
+
 {% block content %}
-<!-- Bookbar -->
-<div class="bookbar">
-  <h1><span class="draft-stat">{% trans "Full View" %}</span>{{ book.title }}</h1>
-</div>
-<!-- end Bookbar -->
 
-<div class="container">
-    <div class="row two-col">
-        <div class="col-xs-12">
-            <div class="box white table_of_contents">
-                <h2 class="box-title">{% trans "Table of Contents" %}</h2>
+  <!-- Bookbar -->
+  <div class="bookbar">
+    <h1>
+      <a class="draft-stat" href="{% url 'reader:infopage' book.url_title %}">{% trans "Info" %}</a>
+      <span class="draft-stat toctoggle closed" id="toctoggle">
```

Comment:
I would change this for an `a` tag or enable `cursor: pointer` to give it more a button feeling :)